charts/xp-management/templates/_helpers.tpl

```diff
@@ -123,33 +123,33 @@ API config related
 {{- end -}}
 
 {{- define "management-svc.defaultConfig" -}}
-port: 8080
-allowedOrigins: "*"
-authorizationConfig:
-  enabled: false
-dbConfig:
-  host: {{ include "common.postgres-host" (list .Values.postgresql .Values.externalPostgresql .Release .Chart ) }}
-  port: 5432
-  database: {{ include "common.postgres-database" (list .Values.postgresql .Values.externalPostgresql .Values.global "mlp" "postgresqlDatabase") }}
-  user: {{ include "common.postgres-username" (list .Values.postgresql .Values.externalPostgresql .Values.global ) }}
-  connMaxIdleTime: {{ .Values.deployment.apiConfig.dbConfig.connMaxIdleTime }}
-  connMaxLifetime: {{ .Values.deployment.apiConfig.dbConfig.connMaxLifetime }}
-  maxIdleConns: {{ .Values.deployment.apiConfig.dbConfig.maxIdleConns }}
-  maxOpenConns: {{ .Values.deployment.apiConfig.dbConfig.maxOpenConns }}
-deploymentConfig:
-  dnvironmentType: dev
-segmenterConfig:
-  s2_ids:
-    minS2CellLevel: 10
-    maxS2CellLevel: 14
-mlpConfig:
-  url: {{ .Values.deployment.apiConfig.mlpConfig.url | default (include "management-svc.mlp.server.url" .) | quote }}
-newRelicConfig:
-  enabled: false
-sentryConfig:
-  enabled: false
-xpUIConfig:
-  appDirectory: /app/xp-ui
+Port: 8080
```

Comment:
@maniteja6799 should we update the values in the values files to match this? https://github.com/caraml-dev/helm-charts/blob/main/charts/xp-management/values.yaml#L30

Reply:
Added.